BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -117,6 +117,8 @@ typedef NSString BOXSharedLinkAccessLevel;
 extern BOXSharedLinkAccessLevel *const BOXSharedLinkAccessLevelOpen;
 extern BOXSharedLinkAccessLevel *const BOXSharedLinkAccessLevelCompany;
 extern BOXSharedLinkAccessLevel *const BOXSharedLinkAccessLevelCollaborators;
+extern BOXSharedLinkAccessLevel *const BOXSharedLinkExpireLinkFileAvailable;
```

Comment:
These aren't quite shared link access levels...
Should these just be separate constants specific to the expiration date?

Reply:
In fact, these actually aren't used anywhere either, can we remove them?

Reply:
removed